src/python/grpcio_tests/tests/fork/_fork_interop_test.py

```diff
@@ -67,8 +65,8 @@ def _dump_streams(name, streams):
 
 
 @unittest.skipUnless(
-    sys.platform.startswith("linux"),
-    "not supported on windows, and fork+exec networking blocked on mac",
+    sys.platform in ("linux", "darwin"),
```

Comment:
This may lose coverage for some linux platforms in CI. See https://docs.python.org/3/library/sys.html#sys.platform